config/sync/core.entity_form_display.node.portfolio.default.yml

```diff
@@ -4,12 +4,19 @@ status: true
 dependencies:
   config:
     - field.field.node.portfolio.body
-    - field.field.node.portfolio.field_picture
+    - field.field.node.portfolio.field_desktop_image
+    - field.field.node.portfolio.field_featured_project_
```

Comment:
Look at the machine name here - it ends in a _. It'd be better for the machine name to be field_featured_project instead of field_featured_project_. This can happen when you type an extra space or a special character (?!) at the end of a field's label. In that case, you'd have to look at the machine name that was generated and correct it before continuing  through the field creation steps.